tezos_interop/fetch_storage.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const fs = require("fs");
+const taquito = require("@taquito/taquito");
+const { RpcClient } = require("@taquito/rpc");
+const { InMemorySigner } = require("@taquito/signer");
+
+const { TezosToolkit } = taquito;
+/**
+ * @typedef Input
+ * @type {object}
+ * @property {string} contract_address
+ * @property {string} rpc_node
+ */
+/** @returns {Input} */
+const input = () => JSON.parse(fs.readFileSync(process.stdin.fd));
+
+/**
+ * @typedef OutputFinished
+ * @type {object}
+ * @property {string} root_hash.current_block_hash
```

Comment:
This output is only the case if the storage is for consensus, if I pass another contract_address here it will not work.